### PR TITLE
fix(block): fix issue with enable a disable block with composer

### PR DESCRIPTION
**Pull request by qsomazzi**

### CODE REVIEW

File: Resources/public/sonata-page.back.css [hunk 1]
```diff
@@ -402,6 +402,15 @@
   margin: 0 !important;
   font-size: 14px !important; }
 
+/*# sourceMappingURL=composer.css.map */
```
Comment (rande) [inline Resources/public/sonata-page.back.css]: You need remove the map

**Comment by qsomazzi** [comment]
@rande done

File: Resources/public/sonata-page.back.css [hunk 2]
```diff
@@ -7,8 +7,8 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  *
- * generated on: Wed Apr 01 2015 00:20:49 GMT-0600 (CST)
```
Comment (rande) [inline Resources/public/sonata-page.back.css]: you can remove this file from the PR

File: Resources/public/sonata-page.front.css [hunk 3]
```diff
@@ -7,8 +7,8 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  *
- * generated on: Tue Mar 24 2015 11:15:49 GMT+0100 (CET)
- * revision:     049fc2564d2cb34e034745ef8074f686bfa8aa60
+ * generated on: Tue Aug 04 2015 10:58:59 GMT+0200 (CEST)
```
Comment (rande) [inline Resources/public/sonata-page.front.css]: you can remove this file from the PR

File: Resources/public/sonata-page.front.js [hunk 4]
```diff
@@ -7,8 +7,8 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  *
- * generated on: Thu Nov 06 2014 00:08:06 GMT+0100 (CET)
```
Comment (rande) [inline Resources/public/sonata-page.front.js]: you can remove this file from the PR

File: Resources/public/sonata-page.front.min.css [hunk 5]
```diff
@@ -7,8 +7,8 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  *
- * generated on: Thu Nov 06 2014 00:08:06 GMT+0100 (CET)
- * revision:     71deefa3b32e338654bad5b904239358daf1d7e6
+ * generated on: Tue Aug 04 2015 10:58:59 GMT+0200 (CEST)
```
Comment (rande) [inline Resources/public/sonata-page.front.min.css]: you can remove this file from the PR

**Comment by qsomazzi** [comment]
@rande done

File: Resources/public/sonata-page.back.min.css [hunk 6]
```diff
@@ -7,8 +7,8 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  *
- * generated on: Wed Apr 01 2015 00:20:49 GMT-0600 (CST)
```
Comment (rande) [inline Resources/public/sonata-page.back.min.css]: This file too ;)

**Comment by qsomazzi** [comment]
@rande last time ? :)

**Pull request merged**